newrelic_lambda_cli/cli/integrations.py

```diff
@@ -163,57 +171,55 @@ def uninstall(aws_profile, aws_region, aws_permissions_check, nr_account_id, for
 @add_options(AWS_OPTIONS)
 @click.option(
     "--enable-logs/--disable-logs",
-    default=False,
+    default=None,
```

Comment:
This will also require a README update I think. Think `default=None` can b omitted?

Reply:
It looks like for boolean flags like this, the default default is False. So we need `default=None` here.